### PR TITLE
Added Match class and solved #24. 

### LINKED ISSUE #24: Points can be given even though matches are not started

### PR DESCRIPTION
**Pull request by frontierpsycho**
Discuss.

Closes #24

### CODE REVIEW

**Comment by frontierpsycho** [comment]
Heh, I'm too hasty. This DOESN'T work in all cases, DON'T MERGE. I'll update later and let you know.

**Comment by frontierpsycho** [comment]
I don't get it :-/ It mostly works correctly, but after I click on "End round", the button disappears.

I checked, though, and the button's condition (`<a v-if="match.isRunning" ...`) is true after I click it. I don't understand at all why the button disappears.

**Comment by frontierpsycho** [comment]
Now it works!

File: js/src/models/Match.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    return m
+  }
+
+  get end () { return this._end }
+  set end (value) { this._end = value }
+
+  get isStarted () {
+    // match is started if 'started' is defined and NOT equal to go's zero date
+    return !isGoZeroDateOrFalsy(this.started)
```
Comment (thiderman) [inline js/src/models/Match.js]: Neat!

**Comment by thiderman** [comment]
This looks really neat! I had no idea about many of the tricks in here, but they look super and they make sense. I had no idea we could do `get ...` and `set ...` stuff in JS!

I've just woken up and I'll take a closer look after a shower and a cup of tea, but initially this looks merge:able! Great job, man!

**Comment by frontierpsycho** [comment]
Glad you like it! :D

**Comment by thiderman** [comment]
I've taken a further look now. Wasn't exactly shortly after my shower, but hey!

This is great! Now I have a good idea of what needs to be done to modernize the code. That is, if you don't want to spearhead that little project! If you want, we can make a new issue about that and you can work on making the js make more sense?

**Comment by frontierpsycho** [comment]
I'd love to! I don't know how consistent I will be with time, though. I can try to work a little bit more and see how well it works, OK?

**Comment by frontierpsycho** [comment]
So, make an issue and assign it to me, and we'll see where it leads.

**Pull request merged**